template/editform_bone_select.html

```diff
@@ -8,7 +8,7 @@
                     name="{{ boneName }}"
                     value="{{ key }}"
                     type="checkbox"
-                    {{ "checked" if key in boneValue }}
+                    {{ "checked" if key in boneValue["key"] }}
```

Comment:
@skoegl Did you test this case (multiple=True)?  boneValue should be a dict[(key: str), (descr: str)] in this case..